### PR TITLE
feat: detail views enhancements

### PR DESCRIPTION
**Pull request by Garfield-fr**
* Closes rero/rero-ils#3618.

### CODE REVIEW

**Comment by PascalRepond** [comment]
## Tests PO

- [ ] REP: Item detailed view: "Local fields" tab label is not translated?
- [ ] REP: Item detailed view: "Borrowed by" box should be hidden if it is empty (cf. before 1.21.0)
- [ ] REP [not-linked but could be fixed in the flow] Item-ISSUE detailed view: the back buttons are weirdly displayed
Now:
image
Before:
image

**Review by PascalRepond — APPROVED** [review]
Great job!

File: projects/admin/src/app/record/detail-view/document-detail-view/document-detail/document-detail.component.html [hunk 1]
```diff
@@ -28,20 +28,29 @@
...
+        }
         @if (!record.metadata.pid && source) {
           <button
             class="btn btn-sm btn-outline-primary ml-1"
             [permissions]="permissions.DOC_CREATE"
             (click)="importDocument($event, record, { source: source, pid: pid })"
-            translate>Import</button>
+          >
+            <i class="fa fa-cloud-download"></i>
+            {{ 'Import' }}
```
Comment (lauren-d) [inline projects/admin/src/app/record/detail-view/document-detail-view/document-detail/document-detail.component.html:41]: Missing `translate` pipe for 'Import'

File: projects/admin/src/app/record/detail-view/holding-detail-view/holding-page-detail/holding-page-detail.component.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import { NgxSpinnerService } from 'ngx-spinner';
+import { ToastrService } from 'ngx-toastr';
+
+@Component({
+  selector: 'admin-holding-page-detail',
+  templateUrl: './holding-page-detail.component.html'
+})
+export class HoldingPageDetailComponent extends DetailComponent {
+
+  get isEnabledOperationLog(): boolean {
```
Comment (lauren-d) [inline projects/admin/src/app/record/detail-view/holding-detail-view/holding-page-detail/holding-page-detail.component.ts:34]: missing docstrings

File: projects/admin/src/app/record/detail-view/item-detail-view/item-page-detail/item-page-detail.component.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+import { NgxSpinnerService } from 'ngx-spinner';
+import { ToastrService } from 'ngx-toastr';
+
+@Component({
+  selector: 'admin-item-page-detail',
+  templateUrl: './item-page-detail.component.html'
+})
+export class ItemPageDetailComponent extends DetailComponent {
+
+  get isEnabledOperationLog(): boolean {
```
Comment (lauren-d) [inline projects/admin/src/app/record/detail-view/item-detail-view/item-page-detail/item-page-detail.component.ts:33]: missing docstrings

**Review by lauren-d — APPROVED** [review]

**Pull request merged**